payments/invoice/seamless-view.md

```diff
@@ -23,107 +23,363 @@ sidebar:
+**Request**
+
+```http
+POST /psp/invoice/payments HTTP/1.1
+Authorization: Bearer <AccessToken>
+Content-Type: application/json
+
+{
+    "payment": {
+        "operation": "FinancingConsumer",
+        "intent": "Authorization",
+        "currency": "SEK",
+        "prices": [
+            {
+                "type": "Invoice",
+                "amount": 4201,
+                "vatAmount": 0
+            }
+        ],
+        "description": "Test Invoice",
+        "userAgent": "Mozilla/5.0...",
+        "generatePaymentToken": false,
+        "generateRecurrenceToken": false,
+        "paymentToken": ""
+        "language": "sv-SE",
+        "urls": {
+            "hosturls": [
+                "http://example.com"
+            ],
+            "completeUrl": "http://example.com/payment-completed",
+            "cancelUrl": "http://example.com/payment-canceled",
+            "callbackUrl": "http://example.com/payment-callback",
+            "termsOfServiceUrl": "http://example.com/payment-terms.pdf"
+        },
+        "payeeInfo": {
+            "payeeId": "12345678-1234-1234-1234-123456789012",
+            "payeeReference": "CD1234",
+            "payeeName": "Merchant1",
+            "productCategory": "A123"
+        }
+    },
+    "invoice": {
+        "invoiceType": "PayExFinancingSE"
+    }
+}
+```
+
+{:.table .table-striped}
+| Required | Property                          | Type          | Description                                                                                                                                                                                                                                                                                                                                         |
+| :------: | :-------------------------------- | :------------ | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+|  ✔︎︎︎︎︎  | `payment`                         | `object`      | The `payment` object contains information about the specific payment.                                                                                                                                                                                                                                                                               |
+|  ✔︎︎︎︎︎  | └➔&nbsp;`operation`               | `string`      | The operation that the `payment` is supposed to perform. The [`FinancingConsumer`][financing-consumer] operation is used in our example. Take a look at the [create `payment` section][create-payment] for a full examples of the following `operation` options: [FinancingConsumer][financing-consumer], [Recur][recur], and [Verify][verify] |
```

Comment:
Why are only a few rows using the heavy check mark, while the rest of the tables does not?